db/seeds/development.rb

```diff
@@ -40,3 +39,13 @@
                access_token_audience: 'va.gov',
                logout_redirect_uri: 'http://localhost:3001',
                refresh_token_duration: SignIn::Constants::RefreshToken::VALIDITY_LENGTH_SHORT_MINUTES)
+
+# Create Config for example external client
+vatemplate = SignIn::ClientConfig.find_or_initialize_by(client_id: 'vatemplate')
+vatemplate.update!(authentication: SignIn::Constants::Auth::COOKIE,
+                   anti_csrf: true,
+                   redirect_uri: 'http://localhost:3001/auth/login/callback',
```

Comment:
I have been using port `4567` to simulate external apps because there are some differences and gotchas when using a port other than `3001`.  It's mainly login.gov config where they restrict form redirections from addresses that aren't whitelisted. Anyway, it's helped us catch and fix some things with SiS
```suggestion
                   redirect_uri: 'http://localhost:4567/auth/result',
```